### PR TITLE
LPS-183871 | Test Fix | Master

### PR DESCRIPTION
**Pull request by davisaints**
**JIRA Ticket: [LPS-183871](https://issues.liferay.com/browse/LPS-183871)**

### CODE REVIEW

**Comment by davisaints** [comment]
ci:test:sf

File: modules/apps/object/object-test/src/testFunctional/tests/ObjectNotifications.testcase [hunk 1]
```diff
@@ -563,7 +563,6 @@ definition {
 	@description = "LPS-178783 - Verify a standalone action can be triggered manually to send an email"
 	@priority = 5
 	test CanSendEmailViaActionStandalone {
-		property custom.properties = "feature.flag.LPS-173894=true";
```
Comment (davisaints) [inline modules/apps/object/object-test/src/testFunctional/tests/ObjectNotifications.testcase:566]: This feature flag was removed. Please [see](https://github.com/brianchandotcom/liferay-portal/pull/132700/commits/ddb19911bf43e531c3861c6c5254ae441bf63e22#diff-e652d04e2647ab1ad8d34df89282488f6088a1b238474173afcca9b01484eb63L6576-L6580)

**Comment by andreefarias** [comment]
Good job @davisaints!
LGTM! 🎉

**Comment by epark18** [comment]
ci:forward

**Pull request closed without merging**